pom.xml

```diff
@@ -146,26 +149,80 @@
+          <configuration>
+            <systemPropertyVariables>
+              <java.util.logging.manager>org.jboss.logmanager.LogManager</java.util.logging.manager>
+              <maven.home>${maven.home}</maven.home>
+              <maven.repo>${settings.localRepository}</maven.repo>
+            </systemPropertyVariables>
+          </configuration>
+        </plugin>
+        <plugin>
+          <artifactId>maven-failsafe-plugin</artifactId>
+          <configuration>
+            <systemPropertyVariables>
+              <java.util.logging.manager>org.jboss.logmanager.LogManager</java.util.logging.manager>
+              <maven.home>${maven.home}</maven.home>
+              <maven.repo>${settings.localRepository}</maven.repo>
+            </systemPropertyVariables>
+          </configuration>
+        </plugin>
+        <!-- Standalone doc artifact, and dry-run for Antora build of https://github.com/quarkiverse/quarkiverse-docs -->
+        <plugin>
+          <groupId>org.asciidoctor</groupId>
+          <artifactId>asciidoctor-maven-plugin</artifactId>
+          <version>${version.asciidoctor.plugin}</version>
+          <configuration>
+            <skip>${skipDocs}</skip>
+            <enableVerbose>true</enableVerbose>
+            <logHandler>
+              <failIf>
+                <severity>WARN</severity>
+              </failIf>
+            </logHandler>
+            <sourceDirectory>${project.basedir}/modules/ROOT/pages/</sourceDirectory>
+            <preserveDirectories>true</preserveDirectories>
+            <attributes>
+              <icons>font</icons>
+              <!-- Antora images path -->
+              <imagesdir>./_images/</imagesdir>
+              <sectanchors>true</sectanchors>
+              <!-- set the idprefix to blank -->
+              <idprefix />
+              <idseparator>-</idseparator>
+              <docinfo1>true</docinfo1>
+              <!-- avoid content security policy violations -->
+              <skip-front-matter>true</skip-front-matter>
+            </attributes>
+          </configuration>
+          <executions>
+            <execution>
+              <id>output-html</id>
+              <phase>prepare-package</phase>
+              <goals>
+                <goal>process-asciidoc</goal>
+              </goals>
+              <configuration>
+                <backend>html5</backend>
+                <attributes>
+                  <source-highlighter>coderay</source-highlighter>
+                  <!-- avoid content security policy violations -->
+                  <linkcss>true</linkcss>
+                  <copycss>true</copycss>
+                </attributes>
+              </configuration>
+            </execution>
+          </executions>
+          <dependencies>
+            <dependency>
+              <groupId>org.asciidoctor</groupId>
+              <artifactId>asciidoctorj</artifactId>
+              <version>${version.asciidoctorj}</version>
+            </dependency>
```

Comment:
Is this overriding the original plugin dependency? Thanks.

Reply:
Yes, it's because the asciidoctor maven plugin doesn't do a release each time there a new version of asciidoctor/asciidoctorj.
So it uses by default a quite old version, whereas Antora probably uses something more recent.

Reply:
Ok, thanks, as long as it's compatible...